packages/cms-base/components/SwProductAddToCart.vue

```diff
@@ -33,7 +33,15 @@ const { product } = toRefs(props);
 const { addToCart, quantity } = useAddToCart(product);
 
 const addToCartProxy = async () => {
-  await addToCart();
+  const res = await addToCart();
+  const errors = Object.values(res.errors);
```

Comment:
Cart errors should be transformed and translated. Please check how it's done in the demo store 

https://github.com/shopware/frontends/blob/main/templates/vue-demo-store/components/product/ProductAddToCart.vue#L12

Reply:
Ah nice thanks! Do we also need to handle any translations or can we rely that the API Response is already translated?

Reply:
Hello :wave:

I have created a new Branch from your PR, and changed the part @mdanilowicz mentioned.

Look here: https://github.com/shopware/frontends/commit/d8e28c8b0eb63976194b4e63e8679c677a34de64

But to be honest I do not like the part about the quantity. Currently, we would need to search in the line items for the quantity that was added to the cart. Maybe a more generic error message would be easier.

Reply:
> Ah nice thanks! Do we also need to handle any translations or can we rely that the API Response is already translated?


Unfortunately, all API errors are in English, even if the context language is different.  So, we have to translate it "locally"